### PR TITLE
Bump node.js to v20

### PR DESCRIPTION
**Pull request by jagodek**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN pip install --no-cache-dir pip install "poetry ~= 1.8.2" \
     && poetry build --format wheel --no-ansi
 
 # Stage 3: Build the application.
-FROM node:18 AS build
+FROM node:19 AS build
```
Comment (grzanka) [inline Dockerfile:30]:
```suggestion
FROM node:20 AS build
```

what about nodejs 20 ? its current LTS version. Does it solve the problem ?

Comment (jagodek) [reply Dockerfile:30]: Unfortunately after recent bios update (another issue with my laptop) sth broke on my linux and graphic interface doesn't work. I'm moving back to windows.

Comment (grzanka) [reply Dockerfile:30]: Then make some check on WSL, it should work, also when you start container from Windows

**Comment by grzanka** [comment]
@jagodek what's the status here ?

**Comment by jagodek** [comment]
@grzanka Another issue. During docker compose poetry is not recognized.

**Comment by grzanka** [comment]
> @grzanka Another issue. During docker compose poetry is not recognized.

Are you in sync with recent master branch?

**Comment by jagodek** [comment]
@grzanka issue solved after prune. Build works with node 20

**Comment by grzanka** [comment]
> @grzanka issue solved after prune. Build works with node 20

Can you commit update to node 20 in this PR

**Comment by grzanka** [comment]
Check as well  if other places require change (GitHub actions)

**Review by grzanka — APPROVED** [review]

**Pull request merged**